en/components/grid.md

```diff
@@ -377,40 +535,178 @@ Defining handlers for these event emitters is done using declarative event bindi
+|`inlineEditorTemplate`|TemplateRef|Yes|Yes|Get/Set a reference to a template which will be applied as a cell enters in edit mode.|
 
 <div class="divider--half"></div>
+
+## Filtering Conditions
+
+You will need to import the appropriate condition types from the `igniteui-angular` package.
+
+    import {
+        STRING_FILTERS,
+        NUMBER_FILTERS,
+        DATE_FILTERS,
+        BOOLEAN_FILTERS
+    } from 'igniteui-angular/main';
+
+### String types
+
+|Name|Signature|Description|
+|--- |--- |--- |
+|`contains`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if the `target` contains the `searchVal`.|
+|`startsWith`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if the `target` starts with the `searchVal`.|
+|`endsWith`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if the `target` ends with the `searchVal`.|
+|`doesNotContain`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if `searchVal` is not in `target`.|
+|`equals`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if `searchVal` matches `target`.|
+|`doesNotEqual`|`(target: string, searchVal: string, ignoreCase?: boolean)`|Returns true if `searchVal` does not match `target`.|
+|`null`|`(target: any)`|Returns true if `target` is `null`.|
+|`notNull`|`(target: any)`|Returns true if `target` is not `null`.|
+|`empty`|`(target: any)`|Returns true if `target` is either `null`, `undefined` or a string of length 0.|
+|`notEmpty`|`(target: any)`|Returns true if `target` is not `null`, `undefined` or a string of length 0.|
+
+<div class="divider--half"></div>
+
+### Number types
+
+|Name|Signature|Description|
+|--- |--- |--- |
+|`equals`|`(target: number, searchVal: number)`|Returns true if `target` equals `searchVal`.|
+|`doesNotEqual`|`(target: number, searchVal: number)`|Returns true if `target` is not equal to `searchVal`.|
+|`doesNotEqual`|`(target: number, searchVal: number)`|Returns true if `target` is greater than `searchVal`.|
+|`lessThan`|`(target: number, searchVal: number)`|Returns true if `target` is less than `searchVal`.|
+|`greaterThanOrEqualTo`|`(target: number, searchVal: number)`|Returns true if `target` is greater than or equal to `searchVal`.|
+|`lessThanOrEqualTo`|`(target: number, searchVal: number)`|Returns true if `target` is less than or equal to `searchVal`.|
+|`null`|`(target: any)`|Returns true if `target` is `null`.|
+|`notNull`|`(target: any)`|Returns true if `target` is not `null`.|
+|`empty`|`(target: any)`|Returns true if `target` is either `null`, `undefined` or `NaN`.|
+|`notEmpty`|`(target: any)`|Returns true if `target` is not `null`, `undefined` or `NaN`.|
+
+<div class="divider--half"></div>
+
+### Boolean types
+
+|Name|Signature|Description|
+|--- |--- |--- |
+|`true`|`(target: boolean)`|Returns if target is truthy.|
```

Comment:
Should `target` here be marked with backticks fro consistency?  Same on line 651.